lxc/restore.go

```diff
@@ -3,39 +3,40 @@ package main
 import (
 	"fmt"
 
+	"github.com/codegangsta/cli"
```

Comment:
Insert a new line after this import to keep it separate from the github.com/lxc/lxd/* ones.